applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_source.h

```diff
@@ -47,13 +47,6 @@ int broadcast_source_stop(void);
  */
 int broadcast_source_send(struct encoded_audio enc_audio);
 
-/**
- * @brief	Enable the LE Audio broadcast (BIS) source.
- *
- * @return	0 for success, error otherwise.
- */
-int broadcast_source_enable(void);
```

Comment:
Note to self, check location of this call.

Reply:
It was duplicated in the unicast_server PR